earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/core/glc_assembler.py

```diff
@@ -29,6 +29,7 @@
 import yaml
 from common import utils
 import common.configs
+from lxml import etree
```

Comment:
Minor: update copyright note to add `Copyright 2019, Open GEE Contributors` for both modified files.

Reply:
Let's create GitHub ticket about this issue if it is not there yet and link it to this PR.